service/src/main/java/greencity/mapping/GetTariffsInfoDtoMapper.java

```diff
@@ -0,0 +1,100 @@
+package greencity.mapping;
+
+import greencity.dto.CourierTranslationDto;
+import greencity.dto.GetTariffsInfoDto;
+import greencity.dto.LocationInfoDto;
+import greencity.dto.LocationTranslationDto;
+import greencity.dto.LocationsDto;
+import greencity.dto.ReceivingStationDto;
+import greencity.dto.RegionTranslationDto;
+import greencity.entity.order.Courier;
+import greencity.entity.order.CourierLocation;
+import greencity.entity.order.TariffsInfo;
+import greencity.entity.user.Location;
+import greencity.entity.user.Region;
+import org.modelmapper.AbstractConverter;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Component
+public class GetTariffsInfoDtoMapper extends AbstractConverter<TariffsInfo, GetTariffsInfoDto> {
+    @Override
+    protected GetTariffsInfoDto convert(TariffsInfo source) {
+        List<CourierLocation> courierLocations = source.getCourierLocations();
+        List<Location> locations = courierLocations.stream()
```

Comment:
Will be better if you write here source source.getCourierLocations().stream() and remove courierLocations;